templates/taxi/car_detail.html

```diff
@@ -18,5 +18,16 @@
     {% for driver in car.drivers.all %}
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
+  {% if user in car.drivers.all %}
+  <form method="post" action="{% url 'car-remove-driver' car.pk user.pk %}">
+    {% csrf_token %}
+    <button type="submit" class="btn btn-danger">Delete me from this car</button>
+  </form>
+{% else %}
+  <form method="post" action="{% url 'car-assign-driver' car.pk user.pk %}">
```

Comment:
Similarly, the URL name 'car-assign-driver' should be verified to ensure it is defined in your `urls.py` for assigning a driver to a car.